weave/tests/test_uploader.py

```diff
@@ -1453,3 +1456,87 @@ def test_upload_from_s3fs(test_basket):
         s3.rm(minio_path,recursive=True)
     if local_fs.exists(pantry_2.index.db_path):
         local_fs.rm(pantry_2.index.db_path)
+
+
+# Skip tests if pymongo is not installed.
+@pytest.mark.skipif(
+    "pymongo" not in sys.modules or not os.environ.get("MONGODB_HOST", False),
+    reason="Pymongo required for this test",
+)
+def test_upload_basket_mongo(test_basket):
+    """Testing pantry.upload_basket(), expected
+
+    to update the collections in mongodb
```

Comment:
Same with above comment, other docstrings in this file keep the description to one/two lines without separating the sentence with a blank line.